polyround.scad

```diff
@@ -165,12 +165,18 @@ let(
 ];
 
 module polyRoundExtrude(radiiPoints,length=5,r1=1,r2=1,fn=10,convexity=10) {
-  orderedRadiiPoints = CWorCCW(radiiPoints) == 1
-    ? reverseList(radiiPoints)
-    : radiiPoints;
+  if(len(radiiPoints) <= 2) {
+    echo(str("There must be at least 3 radii points for polyRoundExtrude. ", radiiPoints, " is not long enough, you need ", 3 - len(radiiPoints), " more point/s. Try:"));

```

Comment:
This should take care of the issue, but what would you think of using an [assertion](https://en.wikibooks.org/wiki/OpenSCAD_User_Manual/The_OpenSCAD_Language#assert) instead to halt rendering at that point and show the message as an error?

Reply:
Oh nice, great tip. I'll make another commit soon.

Reply:
I've update this so now it shows the error.
![image](https://user-images.githubusercontent.com/29681384/120092541-1004d600-c157-11eb-8b1b-33c586b4e230.png)
But I found I still have to wrap parts of the code in `if(len(radiiPoints) <= 2) {` as the assert did not seem to stop it from getting infinite loop 🤷 .